core/unix/os_exports.h

```diff
@@ -45,6 +45,11 @@
 #include <stdarg.h>
 #include "../os_shared.h"
 #include "os_public.h"
+/* arch_exports.h exports opnd.h, but relies on kernel_sigset_t from this header.
```

Comment:
Is this really a circular dependency? arch_exports includes opnd.h, so does os_exports.h (now). Why circular?

Reply:
The comment explains it: arch_exports.h needs kernel_sigset_t from os_exports.h, while os_exports.h needs reg_id_t from arch_exports.h (through opnd.h inside arch_exports.h.